cmd/mavenExecute.go

```diff
@@ -0,0 +1,111 @@
+package cmd
+
+import (
+	"bytes"
+	"github.com/SAP/jenkins-library/pkg/http"
+	"io"
+	"strings"
+
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+)
+
+const mavenExecutable = "mvn"
+
+func mavenExecute(config mavenExecuteOptions, telemetryData *telemetry.CustomData) string {
+	c := command.Command{}
```

Comment:
Just a question: Why do you declare c here instead of inside runMavenExecute?

Reply:
with having command passed as a parameter to runMavenExecute I can easily mock the command (here execMockRunner)

Reply:
Ah, sorry I missed that.